src/web/mjs/engine/I18n.mjs

```diff
@@ -0,0 +1,38 @@
+export default class I18n {
+    constructor() {
+        this.fs = require("fs");
+        this.path = require("path");
+        this.lang =
+            localStorage.getItem("lang") === null
```

Comment:
can simplify default value with ??
`this.lang = localStorage.getItem("lang") ?? "en";`